main.go

```diff
@@ -35,11 +35,12 @@ func main() {
 		return
 	}
 
-	// to enable otel, set OTEL_COLLECTOR_SVC=otel-collector:4317
+	// to enable otel, set OTEL_COLLECTOR_SVC=otel-collector:4317 
```

Comment:
style: there’s an extra whitespace

Reply:
As you mentioned, we will soon have a linter that will catch these. Never mind this.